source/interface/directoryContents.js

```diff
@@ -4,14 +4,12 @@ const { merge } = require("../merge.js");
 const { handleResponseCode, processResponsePayload } = require("../response.js");
 const { normaliseHREF, normalisePath } = require("../url.js");
 const { getSingleValue, getValueForKey, parseXML, propsToStat } = require("./dav.js");
-const { encodePath, prepareRequestOptions, request } = require("../request.js");
+const { appendSlash, encodePath, prepareRequestOptions, request } = require("../request.js");
```

Comment:
No need for `appendSlash` actually.. I discovered that using the `url-join` library is enough:

eg:

```javascript
const joinURL = require("url-join");

joinURL(options.remoteURL, remotePath, "/"); // Creates URL with trailing slash
```